components/reusable-components/login-form/LoginForm.tsx

```diff
@@ -51,11 +52,21 @@ const LoginForm = () => {
     try {
       const response = await submitLoginForm(formValues);
       if (response) {
-        toast.success(response);
+        toast.success('Успешна најава!');
+
+        const userRole = response.role;
+
+        if (userRole === 'content-manager') {
+          window.location.href = '/content-panel';
+        } else if (userRole === 'admin') {
+          window.location.href = '/admin-panel';
+        } else if (userRole === 'user') {
+          window.location.href = '/user-dashboard';
+        }
```

Comment:
Since we are making a call to a route that is intended for login of content manager, we have no use of this logic here.

Reply:
also you have a similar function in utils, **getAuthUrl**